setup.cfg

```diff
@@ -1,11 +1,16 @@
-[aliases]
-test=pytest
-
-
 [metadata]
 license_file = LICENSE
 
 
+[tool:pytest]
+addopts = --cov=yarl -v
+filterwarnings=error
+norecursedirs = dist docs build .tox .eggs venv virtualenv .git
+minversion = 3.8.2
```

Comment:
how about 4.0/5.0?

Reply:
This is a copy-paste from other project.
Please feel free to upgrade to newer versions.